packages/pieces/mysql/src/lib/actions/select-rows.ts

```diff
@@ -0,0 +1,41 @@
+import { createAction, Property } from "@activepieces/pieces-framework";
+import { mysqlCommon, mysqlConnect, isSpecialColumn } from "../common";
+
+export default createAction({
+    name: 'select_rows',
```

Comment:
This is interesting, but it doesn't solve a problem for developers, as most of them will execute a query.

If you aim to make such action no-code friendly, I suggest it be labeled as `Find Row`, and the props Table / Lookup Column / Column Value.

Reply:
Renamed it to Find Rows but kept it plural and also kept the condition as is. I added a description to the condition field to make sure it's clear that it's a sql condition.